packages/docs/src/routes/docs/advanced/prefetching/index.mdx

```diff
@@ -4,25 +4,92 @@ title: Prefetching
 
 # Prefetching
 
-**QUESTION**: _Is lazy loading on user events slow because the user must wait for the code to download?_
+Prefetching is a way for applications to begin downloading modules in a background task, before the user may actually require these modules. The ideal solution is to only prefetch the smallest amount of code that is highly likely to be executed from a user's interaction. Only downloading and executing a minimal amount of JavaScript is an area where Qwik applications excel.
+
+For example, a product page may have a click listener on the "Add to cart" button, and when this button is clicked, the user should immediately get feedback to show the product added to the cart. In this example, Qwik would be able to prefetch the code only for only the event listener, rather than all the JavaScript for the entire application or route. This is in contrast to the traditional approach, where prefetching is used to load the entire application or route, and framework code, just in order for the application to add the click event listener.
```

Comment:
```suggestion
For example, a product page may have a click listener on the "Add to cart" button, and when this button is clicked, the user should immediately get feedback to show the product added to the cart. In this example, Qwik would be able to prefetch the code only for the event listener rather than all the JavaScript for the entire application or route. This is in contrast to the traditional approach, where prefetching is used to load the entire application or route, and framework code, just in order for the application to add the click event listener.
```

Reply:
updated